src/reset.css

```diff
@@ -0,0 +1,121 @@
+* {
+	margin: 0;
+	padding: 0;
+	outline: none;
+	box-sizing: border-box;
+}
+
+article,
+aside,
+details,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+main,
+nav,
+section,
+summary {
+	display: block;
+}
+
+picture,
+audio,
+canvas,
+progress,
+video {
+	display: inline-block;
+	vertical-align: baseline;
```

Comment:
Now styles are reset here for many tags that are not even used on the page, you need to delete unnecessary tags and reset styles only for those that are used on the page